ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -13,60 +13,63 @@
     <form method="post">
         <div asp-validation-summary="ModelOnly"></div>
         <div>
-            <label asp-for="SearchRequest.NhsNumber"></label>
-            <input asp-for="SearchRequest.NhsNumber" />
-            <span asp-validation-for="SearchRequest.NhsNumber"></span>
         </div>
         <div>
             <input type="submit" value="Search" />
         </div>
     </form>
 </div>
-<table>
-    <thead>
-        <tr>
-            <th>
-                @Html.DisplayNameFor(model => model.Results[0].Source)
-            </th>
-            <th>
-                @Html.DisplayNameFor(model => model.Results[0].FamilyNames)
-            </th>
-            <th>
-                @Html.DisplayNameFor(model => model.Results[0].GivenNames)
-            </th>
-            <th>
-                @Html.DisplayNameFor(model => model.Results[0].NhsNumber)
-            </th>
-            <th>
-                @Html.DisplayNameFor(model => model.Results[0].DateOfBirth)
-            </th>
-                        <th>
-                @Html.DisplayNameFor(model => model.Results[0].DateOfNotification)
-            </th>
-        </tr>
-    </thead>
-    <tbody>
-@foreach (var item in Model.Results) {
-        <tr>
-            <td>
-                @Html.DisplayFor(modelItem => item.Source)
-            </td>
-            <td>
-                @Html.DisplayFor(modelItem => item.FamilyNames)
-            </td>
-            <td>
-                @Html.DisplayFor(modelItem => item.GivenNames)
-            </td>
-            <td>
-                @Html.DisplayFor(modelItem => item.NhsNumber)
-            </td>
-            <td>
-                @Html.DisplayFor(modelItem => item.DateOfBirth)
-            </td>
-            <td>
-                @Html.DisplayFor(modelItem => item.DateOfNotification)
-            </td>
-        </tr>
+<div>
+    <div class="clear-search-button-container">
+        <a href="/search" class="clear-search-button">
+            Clear search & start again
+        </a>
+    </div>
+    <div>
+        <p> @Model.Notifications.NumberOfResults Results found. </p>
+        <p> To filter results, add more search terms and search again. </p>
+    </div>
+    <div>
+        @foreach(var result in @Model.SearchResultsBannerDisplay) {
+            <partial name="../Shared/_NotificationBanner.cshtml" model="result"/>
+        }
+    </div>
+
+    <nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">

```

Comment:
I couldn't find the tag helper for pagination but I might have missed it, I'll have another look

Reply:
Looks like the tag helpers can't support asp-page and asp-route-index, is it worth manually coding them into the target url to get the taghelper to work?

Reply:
(just to note, since we talked about it person - yes, let's stil use taghelper. if we can use an asp helper to help build the url great, but if not no big loss either!)